test/thorest-api/transactions/post-call-transaction.test.js

```diff
@@ -0,0 +1,120 @@
+import { ThorWallet, generateAddress } from '../../../src/wallet'
+import { TransactionDataDrivenFlow } from './setup/transaction-data-driven-flow'
+import { successfulCallTxNoRevert, successfulCallTxRevert } from './setup/asserts'
+import { SimpleCounter__factory as SimpleCounter } from '../../../typechain-types'
+
+/**
+ * @group api
+ * @group transactions
+ */
+describe('Call transaction with clauses', function () {
+    const wallet = ThorWallet.withFunds()
+
+    let counter
+
+    beforeAll(async () => {
+        await wallet.waitForFunding()
+        counter = await wallet.deployContract(
+            SimpleCounter.bytecode,
+            SimpleCounter.abi,
+        )
+    })
+
+    it.e2eTest('should simulate vet transfer', 'all', async function () {
```

Comment:
maybe we can add: mixed clauses (like contract call + vet transfer), tx delegation and not enough gas cases. As this should follow the same validation rules